Never use line numbers when running EDITOR.

I'm used to run

    $ nix edit -f. foo
    <edit>
    $ nix edit -f. foo

to get to the location I edited the file previously. Explicit line numer
always resets the position.
--- a/src/libcmd/editor-for.cc
+++ b/src/libcmd/editor-for.cc
@@ -10,7 +10,7 @@ Strings editorFor(const SourcePath & file, uint32_t line)
         throw Error("cannot open '%s' in an editor because it has no physical path", file);
     auto editor = getEnv("EDITOR").value_or("cat");
     auto args = tokenizeString<Strings>(editor);
-    if (line > 0 && (
+    if (false && line > 0 && (
         editor.find("emacs") != std::string::npos ||
         editor.find("nano") != std::string::npos ||
         editor.find("vim") != std::string::npos ||
